bears/java/PMDBear.py

```diff
@@ -0,0 +1,35 @@
+import re
+
+from coalib.bearlib.abstractions.Lint import Lint
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+
+class PMDBear(LocalBear, Lint):
+    executable = 'run.sh'
```

Comment:
at least this won't work definitely under windows^^
In fact this is not that problematic as bears tested only under linux, and under windows we say "known to work", but this is definitely "not known to work" :P